install/kubernetes/cilium/values.yaml

```diff
@@ -1268,3 +1268,11 @@ preflight:
   # Cilium. This will make sure the user will have the policies deployed in the
   # cluster with the right schema.
   validateCNPs: true
+
+# Setting priority class relies on .Capabilities object in our charts.
+# This object is not available for manipulation for `helm template` calls,
+# which causes .Capabilities to be filled according to version of k8s
+# libraries that Helm was compiled against.
+# This option allows to explicitly disable setting the priority class, which
+# is useful for rendering charts for gke clusters in advance.
+disableCriticalPriorityClass: false
```

Comment:
Can this just use a test if gke.enabled and avoid using the flag completely?
This is a double negative. If keeping this, please change the name to enableCriticalPriorityClass and default the value to true.